packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/MultiShareBackupModal/MultiShareBackupModal.tsx

```diff
@@ -0,0 +1,231 @@
+                                    </Paragraph>
+                                    <Checkbox
+                                        isChecked={isChecked1}
+                                        onClick={toggleCheckbox1}
+                                        variant={checkboxVariant1}
+                                    >
+                                        <Translation id="TR_MULTI_SHARE_BACKUP_CHECKBOX_1" />
+                                    </Checkbox>
+                                    <Checkbox
+                                        isChecked={isChecked2}
+                                        onClick={toggleCheckbox2}
+                                        variant={checkboxVariant2}
+                                    >
+                                        <Translation id="TR_MULTI_SHARE_BACKUP_CHECKBOX_2" />
+                                    </Checkbox>
+                                </Section>
+                            </Body>
+                        </>
+                    ),
+                    bottomBarComponents: (
+                        <>
+                            <Button onClick={goToStepNextStep}>
+                                <Translation id="TR_CREATE_MULTI_SHARE_BACKUP" />
+                            </Button>
+                            <LearnMoreButton url={HELP_CENTER_RECOVERY_SEED_URL} size="medium" />
+                            {/*TODO: replace link*/}
+                        </>
+                    ),
+                };
+            case 1:
+                const enterBackup = () => console.log('continue');
+
+                const instructionsSteps: Pick<
+                    BackupInstructionsStepProps,
+                    'children' | 'description' | 'heading' | 'time'
+                >[] = [
+                    {
+                        heading: 'TR_VERIFY_TREZOR_OWNERSHIP',
+                        time: 2,
+                        description: 'TR_VERIFY_TREZOR_OWNERSHIP_EXPLANATION',
+                        children: (
+                            <CardWrapper>
+                                <BackupInstructionsCard isHorizontal icon="BACKUP_2">
+                                    <Translation id="TR_VERIFY_TREZOR_OWNERSHIP_CARD_1" />
+                                </BackupInstructionsCard>
+                                <BackupInstructionsCard isHorizontal icon="CAMERA_SLASH">
+                                    <Translation id="TR_VERIFY_TREZOR_OWNERSHIP_CARD_2" />
+                                </BackupInstructionsCard>
+                            </CardWrapper>
+                        ),
+                    },
+                    {
+                        heading: 'TR_CREATE_SHARES',
+                        time: 10,
+                        description: 'TR_CREATE_SHARES_EXPLANATION',
+                        children: (
+                            <>
```

Comment:
It would be great to somehow separate those two steps to two separate components. This file/component is far too long.

Reply:
5dc5935dd46efe459c6731da34c79c7976ac2a14